WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -698,6 +699,49 @@ public static void setShowGutenbergInfoPopup(String siteURL, boolean show) {
         editor.apply();
     }
 
+    public static boolean isGutenbergInfoPopupDisplayed(String siteURL) {
+        if (TextUtils.isEmpty(siteURL)) {
+            return false;
+        }
+
+        Set<String> urls;
+        try {
+            urls = prefs().getStringSet(DeletablePrefKey.GUTENBERG_OPT_IN_DIALOG_SHOWN.name(), null);
+        } catch (ClassCastException exp) {
+            // no operation - This should not happen.
+            return false;
+        }
+
+        return urls != null && urls.contains(siteURL);
+    }
+
+    public static void setGutenbergInfoPopupDisplayed(String siteURL) {
+        if (isGutenbergInfoPopupDisplayed(siteURL)) {
```

Comment:
Hey @daniloercoli as I can see the method: `isGutenbergInfoPopupDisplayed`
contains code which is already in `setGutenbergInfoPopupDisplayed(String siteURL)`.
WDYT, Can we somehow reuse the code?

Reply:
Yes, I think it's possible to reuse the code, but the refactor is not very straightforward. Going to keep it as-is for now since these checks are going away soon.